schema/image-manifest-schema.json

```diff
@@ -22,7 +22,7 @@
       }
     },
     "annotations": {
-      "id": "https://opencontainers.org/schema/image/manifest-list/annotations",
+      "id": "https://opencontainers.org/schema/image/manifest/annotations",
```

Comment:
This is also [in flight][1] with #484.  I guess whichever way gets the fix in first… ;)

[1]: https://github.com/opencontainers/image-spec/pull/484/files#diff-55ff3e2e7005b8b807f3da93f975e3ebL25